pkg/registry/core/pod/strategy.go

```diff
@@ -130,6 +132,7 @@ func (podStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object)
 
 // WarningsOnUpdate returns warnings for the given update.
 func (podStrategy) WarningsOnUpdate(ctx context.Context, obj, old runtime.Object) []string {
+	// skip warnings on pod update, since the fields we warn about are generally immutable on pods
```

Comment:
i predict this comment will not age well

Reply:
and since humans don't generally interact directly with pods